cleanlab_studio/errors.py

```diff
@@ -38,3 +38,7 @@ class InternalError(Exception):
 
 class CleansetError(InternalError):
     pass
+
+
+class InvalidSchemaTypeError(ValueError):
```

Comment:
these warnings should all show a link to documentation.

would suggest putting that docs link in the `__str__` method of this class. can put a todo for now since we haven't yet written docs 😢